### PR TITLE
Update About page i18n

### LINKED ISSUE #3419: Add i18n to About page

### PR DESCRIPTION
**Pull request by Imperadeiro98**
Also fixed some images
Closes #3419

### CODE REVIEW

File: app/templates/about.jade [hunk 1]
```diff
@@ -178,178 +162,147 @@ block content
...
             .job-listing
-              h5 Sales Representative
+              h5(data-i18n="about.jobs_positon_3_title")
               .text-center
                 small.label
-                  | San Francisco • Fulltime
-              p.small School districts are scrambling to offer computer science classes to all their students as a core subject. They have had no solution, because they can't afford to hire enough programming teachers – until now. 
+                  | San Francisco •
+                  span.spl(data-i18n="about.fulltime")
+              p.small(data-i18n="about.jobs_positon_3_description")
```
Comment (sderickson) [inline app/templates/about.jade]: Thanks for cleaning up the translations! The job position cards are going to be pulled dynamically from lever in the future, though, and so don't need to be translated. Could you remove just those?

**Comment by Imperadeiro98** [comment]
Done

**Comment by sderickson** [comment]
Thanks!

By the way, I'm working on creating a [CodeCombat Cookbook](https://github.com/codecombat/codecombat/wiki/Cookbook), and this morning I added a recipe for [Translating page strings](https://github.com/codecombat/codecombat/wiki/Translate-page-strings). I added the guideline to mark strings with DNT to avoid this problem in the future. Also, can you look over the recipe and let me know if I missed anything for good translation rules?

**Comment by Imperadeiro98** [comment]
I think everything is covered. It should be great to help new people that don't know how to start :+1:

**Pull request merged**